src/Trains.NET.Engine/GamePlay/GameBoard.cs

```diff
@@ -79,15 +79,17 @@ public void GameLoopStep()
         {
             if (!this.Enabled) return;
 
-            _gameUpdateTime.Start();
-            try
-            {
-                DoGameLoopStep();
-            }
-            catch (Exception)
+            using (var _ = _gameUpdateTime.Measure())
             {
+                _gameUpdateTime.Start();
```

Comment:
Why doesn't Measure() call Start()?

Reply:
Haha, it does, I just missed removing this!